src/buildServer.ts

```diff
@@ -96,11 +95,5 @@ export async function buildServer(
   // Endpoints for Cloud Scheduler webhooks (Cron Jobs)
   server.register(routeJobs, { prefix: '/jobs' });
 
-  server.post<{ Params: { service: string } }>(
-    '/slack/:service/webhook',
-    {},
-    SlackRouter(server)
-  );
-
```

Comment:
Why are you removing this ? Isn't this in use by the existing slack integrations  to receive commands a user can click on a message ?

Reply:
No, this was something that I previously added for status page notifications, with the intention of using it for future Slack-related things. But now with this PR, that isn't needed anymore